mlflow/tracking/_model_registry/fluent.py

```diff
@@ -191,7 +191,7 @@ def _register_model(
 
 
 def _get_logged_models_from_run(source_run: str, model_name: str) -> list[LoggedModel]:
-    """Get all logged models from the source rnu that have the specified model name.
+    """Get all logged models from the source run that have the specified model name.
```

Comment:
Let's make this change on master branch. If somebody changes this line on master, that would conflict.

Reply:
Sure, let me make it a separate PR